ports/uni-algo/portfile.cmake

```
@@ -0,0 +1,27 @@
+vcpkg_minimum_required(VERSION 2022-10-12) # for ${VERSION}
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO uni-algo/uni-algo
+    REF "v${VERSION}"
+    SHA512 031d6ec2a1a2c09972a68d7b9bf49a209441e69802d5d8d37b2a37d9b6e002427496d420629d2119dc1d0e80f38c7b220e253b0858db5f172789472447041799
+    HEAD_REF main
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+)
+
+vcpkg_cmake_install()
+
+vcpkg_cmake_config_fixup(NO_PREFIX_CORRECTION)

```

```suggestion
vcpkg_cmake_config_fixup(NO_PREFIX_CORRECTION)

```
`NO_PREFIX_CORRECTION` Currently the correction does not take into account how the files are moved, and applies a rather simply correction which in some cases will yield the wrong results.

---

`vcpkg_cmake_config_fixup` breaks paths in CMake install targets if used without `NO_PREFIX_CORRECTION`.

---

If it breaks, it means that the upstream cmake config file has not been downloaded to the default or specified correct location.

---

If I understand correctly how vcpkg works it will be ON anyway, but I added it just in case.